web/webpack.config.js

```diff
@@ -186,23 +186,23 @@ module.exports = composePlugins(
         });
       }
 
-      if (rule.test.toString().includes("styl")) {
-        const r = rule.oneOf.filter((r) => r.use && r.use.find((u) => u.loader && u.loader.includes("stylus-loader")));
-
-        r.forEach((_r) => {
-          const l = _r.use.filter((u) => u.loader && u.loader.includes("stylus-loader"));
-
-          l.forEach((_l) => {
-            _l.options = {
-              ..._l.options,
-              stylusOptions: {
-                ..._l.options.stylusOptions,
-                import: [path.resolve(__dirname, "apps/labelstudio/src/themes/default/variables.styl")],
-              },
-            };
-          });
-        });
-      }
+      // if (rule.test.toString().includes("styl")) {
+      //   const r = rule.oneOf.filter((r) => r.use && r.use.find((u) => u.loader && u.loader.includes("stylus-loader")));
+
+      //   r.forEach((_r) => {
+      //     const l = _r.use.filter((u) => u.loader && u.loader.includes("stylus-loader"));
+
+      //     l.forEach((_l) => {
+      //       _l.options = {
+      //         ..._l.options,
+      //         stylusOptions: {
+      //           ..._l.options.stylusOptions,
+      //           import: [path.resolve(__dirname, "apps/labelstudio/src/themes/default/variables.styl")],
+      //         },
+      //       };
+      //     });
+      //   });
+      // }
```

Comment:
great job eliminating the need for this 🎉 lets just clear it out instead of commenting it out
```suggestion
```